numpy/core/defchararray.py

```diff
@@ -942,6 +942,20 @@ def isupper(a):
     See Also
     --------
     str.isupper
+
+    Examples
+    --------
+    >>> string = "hello" # lower case
+    >>> np.char.isupper(string)
+    array(False)
+
+    >>> string2 = "HELLO" # upper case
+    >>> np.char.isupper(string2)
+    array(True)
+
+    >>> string3 = "Hello" # mixed case
+    >>> np.char.isupper(string3)
+    array(False)
```

Comment:
For individual strings like this, it would actually be better for users to use the `str.isupper` method. The `np.char.isupper` method is most relevant for arrays of strings. This could be illustrated by collapsing the examples into an array; something like:

```suggestion
    >>> a = np.array(["hello", "HELLO", "Hello"])
    >>> np.char.isupper(a)
    array([False,  True, False])
```